oar/cli/cmd_update_bug_list.py

```diff
@@ -13,8 +13,10 @@
 
 @click.command()
 @click.pass_context
-@click.option("--notify/--no-notify", default=True, help="Send notification to bug owner, default value is true")
-def update_bug_list(ctx, notify):
+@click.option("--notify/--no-notify", default=True, help="Send notification to bug owners, default value is true")
+@click.option("--confirm-droppable", is_flag=True, default=False,
+              help="Send notification only to bug owners with critical and higher issue severity, default value is false")
+def update_bug_list(ctx, notify, confirm_droppable):
```

Comment:
we'd better to add validation for option usage. `--no-notify` should not be used with `confirm-droppable` together, e.g.
```
if not notify and confirm_droppable:
        raise click.UsageError('Error: --no-notify and --confirm-droppable cannot be used together.')
```

Reply:
Updated